x-pack/plugins/fleet/public/components/agent_enrollment_flyout/confirm_agent_enrollment.tsx

```diff
@@ -134,17 +136,31 @@ export const ConfirmAgentEnrollment: React.FunctionComponent<Props> = ({
   if (showLoading && !agentCount) {
     return (
       <>
-        <EuiCallOut
-          size="m"
-          color="primary"
-          iconType={EuiLoadingSpinner}
-          title={
-            <FormattedMessage
-              id="xpack.fleet.agentEnrollment.loading.listening"
-              defaultMessage="Listening for agent..."
-            />
-          }
-        />
+        {isCsp ? (
+          <EuiCallOut
+            size="m"
+            color="primary"
+            iconType={EuiLoadingSpinner}
+            title={
+              <FormattedMessage
```

Comment:
let's only change the `FormattedMessage` component based on the `isCsp`? the rest is the same  and should stay the same in case the callout will be updated for the rest of the Fleet